backend/src/Database.purs

```diff
@@ -71,31 +71,31 @@ withConnection useResource = do
 type DbM m a = ReaderT SQLite.DBConnection (ExceptT Error m) a
 
 execute
-  :: forall m
-   . MonadAff m
-  => SQLite.Query
-  -> Array SQLite.Param
-  -> DbM m Unit
+  ∷ ∀ m
+  . MonadAff m
+  ⇒ SQLite.Query
+  → Array SQLite.Param
+  → DbM m Unit
 execute = query
 
 query
-  :: forall m a
-   . MonadAff m
-  => Decode a
-  => SQLite.Query
-  -> Array SQLite.Param
-  -> DbM m a
+  ∷ ∀ m a
+  . MonadAff m
+  ⇒ Decode a
+  ⇒ SQLite.Query
+  → Array SQLite.Param
+  → DbM m a
 query q params = do
-  conn <- ask
-  f <- lift $ wrap $ map (lmap adaptError) $ liftAff $ try $
+  conn ← ask
+  f ← lift $ wrap $ map (lmap adaptError) $ liftAff $ try $
     SQLite.queryDB conn q params
   lift $ mapExceptT (pure <<< lmap Decoding <<< unwrap) (decode f)
 
-adaptError :: Aff.Error -> Error
+adaptError ∷ Aff.Error → Error
 adaptError e = case Ex.message e of
-  "SQLITE_CONSTRAINT: UNIQUE constraint failed: users.username" ->
```

Comment:
Wow, did we change the SQLite version?

Reply:
Well I don't know, but the thing is it doesn't work without this change